src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final Color color;
+    private final int number;
+
+    Ball(Color color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "Ball " + this.number + " with " + this.color.name().toLowerCase() + " color.";
```

Comment:
You don't need to use the keyword `this` cause you don't have variables with the same name here